src/gwt/src/org/rstudio/studio/client/workbench/views/output/lint/LintManager.java

```diff
@@ -338,15 +337,28 @@ private void showLint(LintContext context, JsArray<LintItem> lint)
       else
          finalLint = lint;
 
-      if (userPrefs_.realTimeSpellchecking().getValue() && TypoSpellChecker.isLoaded())
+      if (userPrefs_.realTimeSpellchecking().getValue())
       {
-         JsArray<LintItem> spellingLint = target_.getSpellingTarget().getLint();
-         for (int i = 0; i < spellingLint.length(); i++)
+         target_.getSpellingTarget().getLint(new ServerRequestCallback<JsArray<LintItem>>()
          {
-            finalLint.push(spellingLint.get(i));
-         }
+            @Override
+            public void onResponseReceived(JsArray<LintItem> response)
+            {
+               for (int i = 0; i < response.length(); i++)
+                  finalLint.push(response.get(i));
+
+               docDisplay_.showLint(finalLint);
+            }
+
+            @Override
+            public void onError(ServerError error)
```

Comment:
We should do _something_ with the error here -- if it isn't appropriate to show it to the user, it should be logged to the debug console at a minimum.